pkg/kong/service.go

```diff
@@ -0,0 +1,79 @@
+package kong
+
+import (
+	"errors"
+	"fmt"
+	"github.com/api7/kong-to-apisix/pkg/apisix"
+	uuid "github.com/satori/go.uuid"
+)
+
+func MigrateService(kongConfig *Config, apisixConfig *apisix.Config) error {
+	kongServices := kongConfig.Services
+	apisixServices := apisixConfig.Services
+
+	for _, kongService := range kongServices {
+		var apisixService apisix.Service
+		apisixService.ID = kongService.ID
+		apisixService.Name = kongService.Name
+		upstreamID := GenerateApisixServiceUpstream(kongService, apisixConfig)
+		apisixService.UpstreamID = upstreamID
+		apisixServices = append(apisixServices, apisixService)
```

Comment:
Better to add a log when we finish migrating a service

Reply:
done